package.json

```diff
@@ -25,10 +28,19 @@
   },
   "homepage": "https://github.com/nextcloud/nextcloud-axios#readme",
   "dependencies": {
-    "@nextcloud/auth": "0.1.0",
-    "axios": "^0.19.0"
+    "@babel/cli": "^7.6.2",
+    "@babel/core": "^7.6.2",
+    "@babel/preset-env": "^7.6.2",
+    "@babel/preset-typescript": "^7.6.0",
+    "@nextcloud/auth": "^0.3.1",
+    "@nextcloud/event-bus": "^0.2.0",
```

Comment:
this should be a dev dependency right?

Reply:
yeah, as long as it's just used in tests